site/content/blogs/release-v116.en.md

```diff
@@ -0,0 +1,180 @@
+# AWS Copilot v1.16: Multiple pipelines, SNS subscription filters and more.
+
+The AWS Copilot core team is excited to announce the Copilot v1.16 release. 
+We welcome 7 new community developers who contributed to this release: [@codekitchen](https://github.com/codekitchen), 
+[@shingos](https://github.com/shingos), [@csantos](https://github.com/csantos), [@rfma23](https://github.com/rfma23), 
+[@g-grass](https://github.com/g-grass), [@isleys](https://github.com/isleys), 
+[@kangere](https://github.com/kangere). Our public [сommunity сhat](https://gitter.im/aws/copilot-cli) is growing and have over 270 people online, 
+who help each other daily and we recently passed a milestone of over 2.1k stars on [Github](http://github.com/aws/copilot-cli/). 
+Thanks to each and every one of you who shows love and support for AWS Copilot.
+
+Copilot v1.16 brings with it several new features and improvements:
+
+* **Multiple pipelines:** You can now run `copilot pipeline init` to create multiple CodePipelines that track separate 
+branches in your repository. **<More details: link within blog to Multiple Pipelines header>**
+* **SNS subscription filter policies:** Worker services can now filter SNS messages for each subscribed topic 
+using the `filter_policy` field. **<More details: links within blog to SNS subscription header>**
+* **Lots of other improvements:**
+    * Add a `--no-rollback` flag to the `deploy` commands to disable automatic stack rollback in case of a deployment failure ([#3341](https://github.com/aws/copilot-cli/pull/3341)). The new flag is useful for troubleshooting infrastructure change failures. For example, if a deployment fails CloudFormation will delete the logs that happened during the failure because it rolls back the stack. This flag will help preserve the logs to troubleshoot the issue, then they can rollback and update their manifest again.
+    * Add a `--upload-assets` flag to the `package` commands to push assets to ECR or S3 before generating CloudFormation templates ([#3268](https://github.com/aws/copilot-cli/pull/3268)). Your pipeline buildspec can now be significantly simplified with this flag. If you'd like to regenerate the buildspec, delete the file and run `copilot pipeline init` again.
```

Comment:
```suggestion
    * Add a `--upload-assets` flag to the `package` commands to push assets to ECR and S3 before generating CloudFormation templates ([#3268](https://github.com/aws/copilot-cli/pull/3268)). Your pipeline buildspec can now be significantly simplified with this flag. If you'd like to regenerate the buildspec, delete the file and run `copilot pipeline init` again.
```

(i'm not 100% certain on the implementation details here, but I thought it was an "and." I may be wrong though :))